Casks/q/qbittorrent.rb

```diff
@@ -1,13 +1,29 @@
 cask "qbittorrent" do
-  on_high_sierra :or_older do
+  on_high_sierra do
```

Comment:
```suggestion
  on_high_sierra :or_older do
```

This is fine to keep as is. Our usual pattern is to set the first block to `:or_older` and, if needed, specify a lower bound using a global `depends_on` (e.g., `depends_on macos: ">= :high_sierra"`).